### ISSUE TITLE
"client" option in "Connection-related options" fails assertion.

### ISSUE THREAD

**Issue by Aryaman1706** [jdesboeufs/connect-mongo#402]
- **I'm submitting a ...**
  [o] bug report
  [ ] feature request
  [ ] question about the decisions made in the repository
  [o] question about how to use this project

- **Summary**
- Assertion is failed on using client option for connection.

- **Other information** 
- Error Message:- "Assertion failed: You must provide either mongoUrl|clientPromise in options"
- Code Snippet:-
```javascript
MongoStore.default.create({
   client: mongoClient,
   mongoOptions: { ... }
})
```

**Comment by mingchuno** [comment]
Ya. My bad. Miss that part.

**Comment by mingchuno** [comment]
Fixed in `4.3.1`

**Comment by Aryaman1706** [comment]
Thanks a lot.
